### DISCUSSION TITLE
Boilerplate YAML Schemas on Project Setup

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3109] [Feature Requests]
This would be part of the initial admin login flow (a pane in that modal) so that these can also pull from external sources (eg: Marketplace).

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
